packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -8,7 +8,9 @@ path: "/design-guidelines/usage-and-behavior/charts"
 _Charts are only available in React_
 
 1. Title: We recommend that titles are always added above and left-aligned to charts.
-2. Legends: We recommend using legends when charts include more than one variable and therefore use more than one color. If space is not available to display names of horizontal and vertical axes, a legend must be added instead. The legend should list the colors that each variable obtains and the name of the variable. They can either be placed left-aligned to the right of the chart or centered below the chart.
+2. Axis labels and scale values: We recommend using tick marks to clearly mark scale values on the y-axis. The text for axis labels and scale values should be 12px font in standard text color. All scale values should be equally distributed across the axis and the axis label should fall outside the scale values, centered with the axis line.
```

Comment:
I think it would really help to have an image to illustrate what you are saying in this section.

Reply:
Okay will add